drivers/hmis/lib/form_data/default/records/move_in_date.json

```diff
@@ -18,6 +18,12 @@
           "id": "max-date",
           "type": "MAX",
           "value_local_constant": "$exitDate"
```

Comment:
Move-in date also can't be after Exit Date. The strictest max bound will be chosen